ast.go

```diff
@@ -2849,6 +2849,12 @@ type ShowTagKeysStatement struct {
 	// Data sources that fields are extracted from.
 	Sources Sources
 
+	// Op to compare tag keys with
+	TagKeyOp Token
+
+	// Literal to compare tag keys with
+	TagKeyExpr Expr
+
```

Comment:
Are `TagKeyOp` and `TagKeyExpr` specific to tags or do they also apply to field keys and expressions? I don't think InfluxQL can know during parsing whether it's a tag or field.

Reply:
It gets re-written post-parsing as `_tagKey <TagKeyOp> <TagKeyExpr>` (or a string of OR expressions instead for WITH KEY IN).